public/index.php

```diff
@@ -190,6 +190,7 @@ function () {
         }
         RenderNewsComponent();
         RenderRecentlyUploadedComponent(5);
+        renderNewClaimsComponent(5);
```

Comment:
I think new achievements (and therefore also new claims) should be _below_ active players. It's not like visitors to the site can engage with them until they're been published.

Reply:
I moved it below the Currently Online widget, I think that and Active Players are related enough where they should be next to each other. The New Achievements widget could probably be updated to new set releases now that it's tracked with claims. Probably needs some additional discussion.